src/Service/DynamoDb/tests/Integration/DynamoDbClientTest.php

```diff
@@ -414,6 +415,6 @@ private function getClient(): DynamoDbClient
 
         return $this->client = new DynamoDbClient([
             'endpoint' => 'http://localhost:8000',
-        ], new Credentials('aws_id', 'aws_secret'));
+        ], new Credentials('aws_id', 'aws_secret'), $httpClient = HttpClient::create());
```

Comment:
I suggest to keep the default client to see if there is something wrong with the retryiableHttpClient

Reply:
I agree. This is just me debugging. I failed to start the docker locally....